telegram/_bot.py

```diff
@@ -44,6 +44,7 @@
 )
 
 from telegram._utils.warnings import warn
+from telegram._utils.warnings_transition import warn_about_thumb_return_thumbnail
```

Comment:
is there a reason why this isn't in the `telegram._utils.warnings` module itself? Seems fitting right?

Reply:
I think I got circular imports error. I state it at the top of the new module if I remember correctly